src/pages/talks/2016-03-building-static-sites-with-react.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import withStyles from 'react-jss';
+
+import Talk from './_talk';
+
+const styles = theme => ({
+  root: {
+    width: '100%'
+  }
+});
+
+const BuildingStaticSites = ({ classes }) => (
+  <Talk>
+    {{
+      title: 'Building Static Sites with React',
+      subtitle: (
+        <div>
+          Ian Sinnott <span>from</span> TruSTAR Technologies
+        </div>
+      ),
+      videoUrl: 'https://www.youtube.com/embed/_1k2HEPH6fY',
+      content: (
+        <div>
+          <p>
+            React.js is an exciting technology for the web that is also making
+            it easier to write native, cross-platform applications. In this
+            talk, Ian Sinnott of TruSTAR Technologies discusses building static,
+            serverless sites using the full power of React.
+          </p>
+          <p>
+            Presented at the March 2016 Real World React meetup at Deco Software
+            in San Francisco, CA.
+          </p>
+        </div>
+      ),
+      specialties: ['react-native', 'graphql', 'aws-lambda', 'microsoft-azure'],
+      nextTalk: {
+        name: 'Building Static Sites with React',
+        url: '/talks/2016-03-building-static-sites-with-react'
```

Comment:
I guess it would be better to use the data coming from the talks data file. Maybe we can try to find the position of the current talk and get the next one on the list. If there is no next one, send null. Keeping in mind that the list should have the items in chronological order.